pkg/status/status.go

```diff
@@ -57,6 +57,16 @@ type Probe struct {
 
 	// OnStatusUpdate is called whenever the status of the probe changes
 	OnStatusUpdate func(status Status)
+
+	// Interval always to specify a probe specific interval that can be
```

Comment:
`s/always/allows` (I think?)

Reply:
if you read "always" fast enough it becomes "allows"